src/main/java/htsjdk/tribble/util/ParsingUtils.java

```diff
@@ -100,16 +101,16 @@ public static InputStream openInputStream(String path)
      * @return
      * @throws IOException
      */
-    public static InputStream openInputStream(String path, Function<SeekableByteChannel, SeekableByteChannel> wrapper)
+    public static InputStream openInputStream(final String path, final Function<SeekableByteChannel, SeekableByteChannel> wrapper)
```

Comment:
Argument `path` should be named `uri` since it's not a path but a URI. Making the caller create a URI would be helpful too.

The javadoc for this could be improved too, it should have full sentences for its description text. And omitting the text for a `@return` or `@throws` is not helpful; either remove the `@` form from the javadoc or add the missing documentation.

Reply:
hmmm. I don't think that changing the API is justified here....I'll develop the docs though